### PR TITLE
Sluggers part 1

### PR DESCRIPTION
**Pull request by ArsTeknica**
## About The Pull Request
<details>
<summary>
Adds in two guns: one a bolt-action survival rifle, and the other a biomatter-based slugthrower.
</summary>
<hr>

	
<hr>
</details>

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
These are in .txt files and are unreadable by BYOND for code

**Comment by ArsTeknica** [comment]
> These are in .txt files and are unreadable by BYOND for code

Fixed.

**Comment by Trilbyspaceclone** [comment]
These need to be in dm
Also they dont complie

**Comment by ArsTeknica** [comment]
Touched it up again. That fix the issues?

File: code/modules/projectiles/guns/projectile/boltgun/london [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/obj/item/gun/projectile/boltgun/london
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun/london:1]: 2 same file

File: code/modules/projectiles/guns/projectile/boltgun/london.dm [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/obj/item/gun/projectile/boltgun/london
+	name = "\"London"\" Survival Rifle"
+	desc = "A single-shot combination rifle, initially based off shipside varmit guns. While almost certainly underpowowered for the local fauna,  it's decent with the scope, and  one of the cheapest longarms around. Chambered in .40 Magnum, with an underslung shotgun barrel.
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun/london.dm:3]: Dosnt complie

File: icons/obj/guns/projectile/kodiak.dm [hunk 3]
```diff
@@ -0,0 +1,24 @@
+obj/item/gun/projectile/shotgun/Kodiak
```
Comment (Trilbyspaceclone) [inline icons/obj/guns/projectile/kodiak.dm:1]:
```suggestion
/obj/item/gun/projectile/shotgun/Kodiak

```

**Comment by Kazkin** [comment]
:V lets make sure this works before merging, I'd do it now but I just spent 8 hours grinding steel.

**Comment by Kazkin** [comment]
This Pr has alot of issues, the kodiak sprite meets our standards of spriting, but the london is unfortunately too poor to be merged. That sprtite will need to be fixed/updated.

**Comment by ArsTeknica** [comment]
I see. Any issues in particular?

File: code/modules/projectiles/guns/projectile/boltgun/london.dm (hunk 2 above)
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun/london.dm:3]:
```suggestion
	desc = "A single-shot combination rifle, initially based off shipside varmit guns. While almost certainly underpowowered for the local fauna,  it's decent with the scope, and  one of the cheapest longarms around. Chambered in .40 Magnum, with an underslung shotgun barrel."

```

File: code/modules/projectiles/guns/projectile/boltgun/london.dm [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	name = "\"London"\" Survival Rifle"
+	desc = "A single-shot combination rifle, initially based off shipside varmit guns. While almost certainly underpowowered for the local fauna,  it's decent with the scope, and  one of the cheapest longarms around. Chambered in .40 Magnum, with an underslung shotgun barrel.
+	icon = 'icons/obj/guns/projectile/london.dmi'
+	icon_state = "london"
+	item_state = "london"
+	slot_flags = SLOT_BELT|SLOT_BACK
+	force = WEAPON_FORCE_PAINFUL
+	caliber = CAL_MAGNUM
+	max_shells = 1
+zoom_factor= 1.3
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun/london.dm:11]:
```suggestion
	zoom_factor= 1.3

```

File: code/modules/projectiles/guns/projectile/boltgun/london.dm [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+	price_tag = 300
+	recoil_buildup = 10
+	damage_multiplier = 0.9 // The Clarissa of boltguns
+	penetration_multiplier  = 1.5
+	extra_damage_mult_scoped = .8 // better if scoped.
+	matter = list(MATERIAL_STEEL = 15, MATERIAL_WOOD = 20,)
+	saw_off = FALSE
+        init_firemodes = list(
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun/london.dm:19]:
```suggestion
       init_firemodes = list(

```

File: code/modules/projectiles/guns/projectile/boltgun/london.dm [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+	slot_flags = SLOT_BELT|SLOT_BACK
+	force = WEAPON_FORCE_PAINFUL
+	caliber = CAL_MAGNUM
+	max_shells = 1
+zoom_factor= 1.3
+	price_tag = 300
+	recoil_buildup = 10
+	damage_multiplier = 0.9 // The Clarissa of boltguns
+	penetration_multiplier  = 1.5
+	extra_damage_mult_scoped = .8 // better if scoped.
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun/london.dm:16]:
```suggestion
	extra_damage_mult_scoped = 0.8 // better if scoped.

```

File: code/modules/projectiles/guns/projectile/boltgun/london [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	max_shells = 1
+zoom_factor= 1.3
+	price_tag = 300
+	recoil_buildup = 10
+	damage_multiplier = 0.9 // The Clarissa of boltguns
+	penetration_multiplier  = 1.5
+	extra_damage_mult_scoped = .8 // better if scoped.
+	matter = list(MATERIAL_STEEL = 15, MATERIAL_WOOD = 20,)
+	saw_off = FALSE
+        init_firemodes = list(
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun/london:19]:
```suggestion
	init_firemodes = list(
```

File: code/modules/projectiles/guns/projectile/boltgun/london (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/boltgun/london:1]: Delete this file its not even a dm

**Comment by Trilbyspaceclone** [comment]
With that file remove any gun issues with this can be sloved my be in a later state as well as adding them to loot pools/disks

**Comment by Kazkin** [comment]
One of the sprites still fails to meet out sprite standards, sadly.

**Pull request closed without merging**